### PR TITLE
Minor grammar fixes

### PR DESCRIPTION
**Pull request by bngsudheer**
Spacing and capitalization

### CODE REVIEW

File: docs/project_structure.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ First we need to install pyramid framework itself::
 
     pip install pyramid==1.5.7
 
-This will install pyramid itself with it's base dependencies, your python 
+This will install pyramid itself with it's base dependencies, your Python
```
Comment (stevepiercy) [inline docs/project_structure.rst]: While you're at it, would you please make a couple more commits? In this context "it's" should be "its". "It's" is the contraction for "it is", whereas "its" is a possessive pronoun.

File: docs/project_structure.rst [hunk 2]
```diff
@@ -38,7 +38,7 @@ We will end up with pyramid_blogr dir that should have following structure::
 Adding dependencies to the project
 ----------------------------------
 
-Since pyramid tries it's best to be a non-opinionated solution we will have to 
+Since Pyramid tries it's best to be a non-opinionated solution we will have to
```
Comment (stevepiercy) [inline docs/project_structure.rst]: s/its/it's

**Comment by bngsudheer** [comment]
Yes, 'its' fix is in another commit. Will send a few more later today or tomorrow.

**Comment by ergo** [comment]
thank you

**Pull request merged**